src/components/auth/auth.resolver.ts

```diff
@@ -47,4 +52,21 @@ export class AuthResolver {
     await this.authService.logout(session.token);
     return true;
   }
+
+  @Mutation(() => Boolean, {
+    description: 'Forgot password; send password reset email',
+  })
+  async forgetPassword(@Args('email') email: string): Promise<boolean> {
+    if (await this.userService.checkEmail(email)) return false;
```

Comment:
It would be better if this check was in `AuthService`. We want all business logic in services. Resolvers are only the transport medium.